introduction.md

```diff
@@ -0,0 +1,215 @@
+
+## Comments
+
+Comments can explain what the code is doing, leave instructions for developers using the code, or add any other useful annotations.
+
+There are two types of code comments in JavaScript:
+
+- A single line comment will comment out a single line and is denoted with two forward slashes ```//``` preceding it.
+
+```
+// Prints 5 to the console
+console.log(5);
+```
+
+Single line comment can also be used to comment after a line of code:
+
+```
+console.log(5);  // Prints 5 
+```
+
+- A multi-line comment will comment out multiple lines and is denoted with ```/*``` to begin the comment, and ```*/``` to end the comment.
+
+```
+/*
+This is all commented 
+console.log(10);
+None of this is going to run!
+console.log(99);
+*/
+```
+
+This syntax can also be used to comment something out in the middle of a line of code:
+
+```
+console.log(/*IGNORED!*/ 5);  // Still just prints 5
+```
+
+## Data Types
+
+Data types are the classifications we give to the different kinds of data that we use in programming. In JavaScript, there are seven fundamental data types:
+
+- Number: Any number, including numbers with decimals: ```4```, ```8```, ```1516```, ```23.42```.
+
+- String: Any grouping of characters on your keyboard (letters, numbers, spaces, symbols, etc.) surrounded by single quotes: ```' ... '``` or double quotes ```" ... "```. Though we prefer single quotes. Some people like to think of string as a fancy word for text.
+
+- Boolean: This data type only has two possible values— either ```true``` or ```false``` (without quotes). It’s helpful to think of booleans as on and off switches or as the answers to a “yes” or “no” question.
+
+- Null: This data type represents the intentional absence of a value, and is represented by the keyword ```null``` (without quotes).
+
+- Undefined: This data type is denoted by the keyword ```undefined``` (without quotes). It also represents the absence of a value though it has a different use than ```null```.
+
+- Symbol: A newer feature to the language, symbols are unique identifiers, useful in more complex coding.
+
+- Object: Collections of related data.
+
+
+The first 6 of those types are considered primitive data types. They are the most basic data types in the language. Objects are more complex.
+
+
+## Arithmetic Operators
+
+An operator is a character that performs a task in our code. JavaScript has several built-in in arithmetic operators, that allow us to perform mathematical calculations on numbers. These include the following operators and their corresponding symbols:
+
+- Add: ```+```
+
+- Subtract: ```-```
+
+- Multiply: ```*```
+
+- Divide: ```/```
+
+- Remainder: ```%```
+
+
+```
+console.log(3 + 4); // Prints 7
+console.log(5 - 1); // Prints 4
+console.log(4 * 2); // Prints 8
+console.log(9 / 3); // Prints 3
+console.log(11 % 3); // Prints 2
+```
```

Comment:
Use `javascript` keyword here so the markdown knows your code is in JS. Doing this will make the code look like this looks like this:

```javascript
console.log(3 + 4); // Prints 7
console.log(5 - 1); // Prints 4
console.log(4 * 2); // Prints 8
console.log(9 / 3); // Prints 3
console.log(11 % 3); // Prints 2
```

instead of this

```
console.log(3 + 4); // Prints 7
console.log(5 - 1); // Prints 4
console.log(4 * 2); // Prints 8
console.log(9 / 3); // Prints 3
console.log(11 % 3); // Prints 2
```


```suggestion
```javascript
console.log(3 + 4); // Prints 7
console.log(5 - 1); // Prints 4
console.log(4 * 2); // Prints 8
console.log(9 / 3); // Prints 3
console.log(11 % 3); // Prints 2
```
```